app/views/users/single_market.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>
+  <% market = @market %>
```

Comment:
Thank you! ^_^

Sent from my iPhone

> On May 3, 2016, at 3:10 PM, Jessica Noglows <notifications@github.com> wrote:
> 
> In app/views/users/single_market.html.erb:
> 
> > @@ -0,0 +1,40 @@
> > +<h1>
> > +  <% market = @market %>
> This line is unnecessary. You could just use the instance variable in the below places that you have "market"
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>